share/spice/movie/movie.js

```diff
@@ -90,37 +81,99 @@
                 }
             },
             relevancy: {
-                skip_words: ['movie', 'info', 'film', 'rt', 'rotten', 'tomatoes', 'rating', 'ratings', 'rotten'],
-                primary: [{
-                    key: 'title'
-                }]
+                skip_words: ['movie', 'info', 'film', 'rt', 'rotten', 'tomatoes', 'rating', 'ratings', 'rotten', 'cast', 'of'],
+                //primary: [{ // TODO: Need to fix this
+                //    key: 'title'
+                //}]
             },
-            onItemShown: function(item) {
-                if (!item.alternate_ids || !item.alternate_ids.imdb) { return; }
+            onItemShown: function (item) {
+                isCastQuery ? castPicture(item) : moviePicture(item);
+            }
+        });
+        function normalizeMovie(item) {
```

Comment:
Let's define all these private functions _outside_ our callback at the top, along with `getImage`